static/styles/compose.css

```diff
@@ -250,6 +250,11 @@
         &:hover {
             opacity: 1;
         }
+
```

Comment:
Why do we need this change? We want outlines on `:focus` to help keyboard users.

Reply:
Oh. We were getting the outlines on clicking too. I and @alya thought its a bug. Will remove this change then. Is there a way to hide it on clicks? Do we need an outline for clicks too?

Reply:
Removed the commit adding this.

Reply:
The compose box formatting buttons have outlines with the keyboard but not when you click on them, and I think that's the behavior we want.

Reply:
Using `a` tag instead of `button` here should solve this.

Reply:
Yes, that solved this, thanks!